ivy/functional/frontends/paddle/tensor/math.py

```diff
@@ -1,5 +1,5 @@
 # local
-from ..math import *  # noqa: F401
+from ..math import exp, floor, lerp, reciprocal, sqrt  # noqa: F401
```

Comment:
I would not change this line, keep the changes to the functions that you contribute to.